.craft.yml

```diff
@@ -2,20 +2,16 @@ minVersion: '0.23.1'
 changelogPolicy: simple
 preReleaseCommand: bash scripts/craft-pre-release.sh
 targets:
-  #
-  # Deactivated for now. This needs to be reactivated if the new Sentry Lambda Extension is deployed to production.
-  # (ask Anton Pirker if you have questions.)
-  #
-  # - name: aws-lambda-layer
-  #   includeNames: /^sentry-node-serverless-\d+.\d+.\d+(-(beta|alpha)\.\d+)?\.zip$/
-  #   layerName: SentryNodeServerlessSDK
-  #   compatibleRuntimes:
-  #     - name: node
-  #       versions:
-  #         - nodejs10.x
-  #         - nodejs12.x
-  #         - nodejs14.x
-  #   license: MIT
+  - name: aws-lambda-layer
+    includeNames: /^sentry-node-serverless-\d+.\d+.\d+(-(beta|alpha)\.\d+)?\.zip$/
+    layerName: SentryNodeServerlessSDK
+    compatibleRuntimes:
```

Comment:
realize craft doesn't support this -  but Lambda has `Compatible architectures` specs, `x86` or `arm64`

from sentry docs (rather hidden), seems like arm64 is not support. is this still true?

Reply:
Arm Lambda functions will be supported in a later release.